X_tip_body:
  x: 0.0107577
  y: 0.0123065
  z: 0.662619
X_tcp_cage:
  x: 0
  y: 0
  z: -0.123926
hover_above_tip: 0.08
insert_below_tip: 0.03
topple:
  forward: 0.2
  downward: 0.03
zigzag:
  step_forward: 0.1
  step_sideways: 0.1
  cycles: 5
  max_vel: 1
  max_acc: 0.5